libdnf/libdnf.pc.in

```diff
@@ -5,7 +5,6 @@ includedir=${prefix}/include
 Name: libdnf
 Description: Simple package manager interface using libsolv and librepo
 Version: @LIBDNF_VERSION@
-Requires: glib-2.0, librepo, rpm, sqlite3
```

Comment:
Why is `sqlite3` being removed?

Reply:
Because none of the header files are including sqlite headers -> to compile anything which is using libdnf ABI/API yoo don't need to have installed sqlite devel resources.

Reply:
Okay...

Reply:
@kloczek I'm not entirely sure how you missed it (in the linked issue), but `sqlite3` is certainly used, have a look at `libdnf/utils/sqlite3/Sqlite3.{c,h}pp`. (this is the wrong commit, btw., just commenting where the conversation is)

Reply:
Oh, but it's not present in public headers (which I suppose is what you grepped). So, shouldn't it go to `Requires.private`?